apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/FakeStorageService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Google.Apis.Storage.v1;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Google.Apis.Requests;
+using Google.Apis.Util;
+using Google.Cloud.ClientTesting;
+using System.Net;
+using System.Net.Http;
+
+namespace Google.Cloud.Storage.V1.Tests
+{
+    class FakeStorageService : StorageService
+    {
+        // TODO: Vary this content based on the actual error code?
```

Comment:
I think it would be worth turning this into a GetErrorContent method that changed the "code" part (and had a general message) so that it's not confusing when it shows up in a failure for a non-401 status code.